packages/redis/src/index.d.ts

```diff
@@ -19,7 +19,7 @@ declare class KeyvRedis extends EventEmitter implements Store<string | undefined
 	has(key: string): boolean;
 }
 declare namespace KeyvRedis {
-	interface Options extends ClientOpts {
+	interface Options {
```

Comment:
@jaredwray, I believe `@keyv/redis@2.3.3` is supposed to have this change, but it doesn't:

```
> tsc -p tsconfig.json

node_modules/@keyv/redis/src/index.d.ts:22:28 - error TS2304: Cannot find name 'ClientOpts'.

22  interface Options extends ClientOpts {
                              ~~~~~~~~~~


Found 1 error in node_modules/@keyv/redis/src/index.d.ts:22



$ grep @keyv/redis package.json 
    "@keyv/redis": "^2.3.3",
```